frontend/src/components/submission/SubmissionForm.vue

```diff
@@ -39,9 +40,19 @@ const assigneeOptions: Ref<Array<User>> = ref([props.submission.user]);
 // Default form values
 const initialFormValues: any = {
   ...props.submission,
+  applicationStatus: props.submission.applicationStatus
+    ? props.submission.applicationStatus
+    : APPLICATION_STATUS_LIST.NEW,
```

Comment:
i think this can be removed now that youre adding NEW during chefs data import(?)